pkg/ui/v1alpha3/frontend/src/components/HP/Create/HPParameters.jsx

```diff
@@ -128,7 +130,7 @@ const HPParameters = (props) => {
             goTemplate: {
                 templateSpec: {
                     configMapName: "trial-template",
-                    configMapNamespace: data.metadata.namespace,
+                    configMapNamespace: deploymentNamespace,
```

Comment:
Is this correct? Can't the template be outside kubeflow namespace?

Reply:
I think the config map should be kept in kubeflow namespace. We have some built-in templates. If we use data.metadata.namespace, we cannot get them if users use other namespaces.